### ISSUE TITLE
Undefined property of Stripe\Account instance: support_email

### ISSUE THREAD

**Issue by dahomz** [stripe/stripe-php#228]
Hi, I'm receiving this error when attempting to access the 'support_email' property of the Stripe account as suggested here https://stripe.com/docs/api#account_object

For the test account I can see this is populated within stripe, just don't seem to be able to access it (using 3.9.0)

**Comment by matthewarkin** [comment]
@dahomz can you provide a snippet of the code you're using?

**Comment by brandur** [comment]
It does look like this is indeed missing by default:

``` php
<?php

require_once('vendor/autoload.php');

\Stripe\Stripe::setApiKey("sk_test_REDACTED");

$account = \Stripe\Account::retrieve("acct_1032D82eZvKYlo2C");
echo $account->support_email;

?>
```

Produces:

``` sh
$ php test.php
Stripe Notice: Undefined property of Stripe\Account instance: support_email
```

I'm just following up with our Connect team to try and verify what the expected behavior here is.

**Comment by brandur** [comment]
Hi @dahomz,

I checked on this one a bit, and it seems that this is mostly a documentation problem. The `support_email` field only comes back for an account where it's not null, and it's only not null when set on a managed account _or_ for your own account.

I'm closing this out for now, but I've added a task internally to loop around and see if we can can make the documentation more clear. Sorry about the trouble!
